src/ophys_etl/transforms/suite2p_container.def

```diff
@@ -0,0 +1,44 @@
+Bootstrap: docker
+From: continuumio/miniconda3:4.8.2
+
+%post
+    # these can be set during build like
+    # sudo SINGULARITYENV_SUITE2P_TAG=0.7.5 singularity build ...
+    SUITE2P_TAG=${SUITE2P_TAG:-0.7.5}
+    OPHYS_ETL_TAG=${OPHYS_ETL_TAG:-master}
+
+    export PATH=/opt/conda/bin:${PATH}
+
+    # initialize conda
+    CONDA_BASE=$(conda info --base)
+    . $CONDA_BASE/etc/profile.d/conda.sh
+
+    # make this variable available at runtime
+    echo "export CONDA_BASE=\"${CONDA_BASE}\"" >> $SINGULARITY_ENVIRONMENT
+
+    # install Suite2P
+    apt-get install -q -y libgl1-mesa-glx
+    git clone https://github.com/MouseLand/suite2p
+    cd suite2p
+    git checkout ${SUITE2P_TAG}
+    conda env create -q -f environment.yml
+    conda activate suite2p
+    pip install -q .
+
+    # install ophys_etl_pipelines
+    cd ../
+    git clone https://github.com/AllenInstitute/ophys_etl_pipelines
+    cd ophys_etl_pipelines
+    git checkout ${OPHYS_ETL_TAG}
+    pip install -q .
+
+    # clean up
+    conda clean -q --all
+    cd ../
+    rm -rf suite2p
+    rm -rf ophys_etl_pipelines
```

Comment:
I'll have to look through the singularity docs again, but generally you want to perform these clean up steps in one command with the setup steps to no create unnecessary cached build steps (see: https://pythonspeed.com/articles/docker-caching-model/)

Reply:
There is a multi-stage build described [here](https://sylabs.io/guides/3.5/user-guide/definition_files.html#multi-stage-builds) but in a quick test of trying to copy over the built conda env, it reduced the image size only 50M _and_ broke the container. So, it seems like more work to figure that out with pretty low potential payoff.